index.bs

```diff
@@ -2671,6 +2687,70 @@ credential. It is intended primarily for [RPS] that wish to tightly control the
+
+This section registers the extension identifiers defined in Section [[#extensions]] in the
+IANA "WebAuthn Extension Identifier" registry established by [[WebAuthn-Registries]].
+
+- WebAuthn Extension Identifier: appid
+- Description: This authentication extension allows [RPS] who have previously registered a credential using the legacy FIDO JavaScript APIs to request an assertion.
+- Specification Document: Section [[#extension-appid]] of this specification
+
+- WebAuthn Extension Identifier: txAuthSimple
+- Description: This signature extension allows for a simple form of transaction authorization. A WebAuthn Relying Party can specify a prompt string, intended for display on a trusted device on the authenticator
+- Specification Document: Section [[#extension-txauth-simple]] of this specification
+
+- WebAuthn Extension Identifier: txAuthGeneric
+- Description: This generic txauth extension allows images to be used as prompts as well. This allows authenticators without a font rendering engine to be used and also supports a richer visual appearance than accomplished with the webauthn.txauth.simple extension. 
+- Specification Document: Section [[#extension-txauth-generic]] of this specification
+
+- WebAuthn Extension Identifier: authnSel
+- Description: This registration extension allows a WebAuthn Relying Party to guide the selection of the authenticator that will be leveraged when creating the credential. It is intended primarily for WebAuthn Relying Parties that wish to tightly control the experience around credential creation.
+- Specification Document: Section [[#extension-authenticator-selection]] of this specification
+
+- WebAuthn Extension Identifier: exts
+- Description: The Supported Extensions extension data is a list (CBOR array) of extension identifiers encoded as UTF-8 Strings. This extension is added automatically by the authenticator. This extension can be added to attestation statements.
+- Specification Document: Section [[#supported-extensions-extension]] of this specification
+
+- WebAuthn Extension Identifier: uvi
+- Description: The user verification index (UVI) is a value uniquely identifying a user verification data record. The UVI data can be used by servers to understand whether an authentication was authorized by the exact same biometric data as the initial key generation. This allows the detection and prevention of "friendly fraud". 
+- Specification Document: Section [[#uvi-extension]] of this specification
+
+- WebAuthn Extension Identifier: loc
+- Description: The location extension provides the client device's current location to the WebAuthn relying party, if supported by the client device and subject to user consent. 
+- Specification Document: Section [[#uvi-location]] of this specification
+
+- WebAuthn Extension Identifier: uvm
+- Description: The user verification mode (UVM) extension returns to the Webauthn relying party which user verification methods (factors) were used for the WebAuthn operation. 
```

Comment:
I believe this is both a registration extension and authentication extension, yes?